extra/dist/tarantoolctl.in

```diff
@@ -25,6 +25,17 @@ int getppid(void);
 
 local TIMEOUT_INFINITY = 100 * 365 * 86400
 
+-- Use getenv_safe when it's present. When not, fall back to plain getenv.
+-- getenv_safe accepts name and size, getenv accepts only name. So we can call
+-- both functions with same arguments. Plain getenv will simply ignore the extra
+-- argument.
+local getenv = os.getenv_safe or os.getenv
+-- How many digits there are in a 8 byte integer + 1 digit for null terminator.
+-- UIDs and GIDs actually use 4 byte integers, but better be on the safe side.
+local UINT64_STRSIZE_MAX = 21
```

Comment:
Why `UINT64`? Should be called `UUID_STRSIZE_MAX`? BTW we have `UUID_STR_LEN` constant in C, which equals 36.

Reply:
It's not about UUID, its UID (user id from linux). As far as I could google, modern linux supports 4 byte uids (that'd use up to 10 digits in string representation). I decided to allow 8 byte ones as well, so that we don't have to go back and patch this place once someone complains about his 8-byte uid not being accepted.